components/browser/session/src/main/java/mozilla/components/browser/session/Session.kt

```diff
@@ -98,6 +100,7 @@ class Session(
         fun onFindResult(session: Session, result: FindResult) = Unit
         fun onDesktopModeChanged(session: Session, enabled: Boolean) = Unit
         fun onFullScreenChanged(session: Session, enabled: Boolean) = Unit
+        fun onMetaViewportFitChanged(session: Session, layoutInDisplayCutoutMode: Int) = Unit
```

Comment:
Since we're not going the enum route, we should probably document what this int value references and the supported values.

Reply:
Agreed.  I'll like to Android docs